test/bdd/stress_steps.go

```diff
@@ -381,11 +384,16 @@ func (r *createDIDReq) Invoke() (interface{}, error) {
 		return nil, err
 	}
 
-	logger.Infof("created did successfully %s", intermID)
+	createLogCount++
+
+	if createLogCount%100 == 0 {
```

Comment:
Need to use atomic.LoadInt64.

Reply:
Actually, you don't need to load - just use the new value returned from Add.

Reply:
done